alvr/client_openxr/src/extra_extensions/body_tracking_bd.rs

```diff
@@ -205,14 +286,92 @@ impl BodyTrackerBD {
             destroy_body_tracker,
             locate_body_joints,
             get_body_tracking_state,
+            get_motion_tracker_connect_state,
+            get_motion_tracker_locations,
+            get_motion_tracker_mode,
         })
     }
 
+    pub fn locate_motion_trackers(
+        &self,
+        time: xr::Time,
+    ) -> xr::Result<Vec<MotionTrackerLocationsBD>> {
```

Comment:
Should this be moved to another struct? it seems it's not using the XrBodyTrackerBD handle

Reply:
Where should it be moved to then?

Reply:
Make another class, or maybe just plain functions, because it doesn't rely on persistent state or handles

Reply:
Maybe I'm not understanding this fully, you're calling `get_motion_tracker_mode`, who sets the tracking mode?

Reply:
I see. So what happens if the user selects motion tracking mode but we poll skeleton data?

Reply:
Body joints will stop updating and return last tracked location.

Reply:
I see. not the best but it's workable. So i'd say let go ahead and split off a structure to manage the motion tracking. Make a separate file for it.